plugins/module_utils/core.py

```diff
@@ -533,6 +518,16 @@ def update_resource(
 
         return results
 
+    def get_identifier(self, identifier: dict, primary_identifier: list ):
+        for id in primary_identifier:
+            if id == "acl_name":
+                identifier["ACLName"] = self.module.params.get("acl_name")
+            else:
+                identifier[
+                    snake_to_camel(id, capitalize_first=True)
+                ] = self.module.params.get(id)
+        return json.dumps(identifier)
+
 
```

Comment:
Can you please check if the update works too? I'm not sure if we need to add something here https://github.com/ansible-collections/amazon.cloud/blob/main/plugins/module_utils/core.py#L467 too.

Reply:
Though the update happens as expected, the scrub_key is not scrubbing ACLName. So I have added a fix. Let me know what you think.